packages/hash/frontend/src/blocks/page/BlockHighlight.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTheme } from "@mui/material";
+import { motion } from "framer-motion";
+import { useBlockHighlightContext } from "./BlockHighlightContext";
+import { useBlockView } from "./BlockViewContext";
+
+interface HighlightProps {
+  onAnimationComplete: () => void;
+}
+
+const Highlight = ({ onAnimationComplete }: HighlightProps) => {
+  const theme = useTheme();
+
+  return (
+    <motion.div
```

Comment:
Can we use MUI's [Fade](https://mui.com/material-ui/api/fade/) component for this instead of framer's `motion` component. We only make use of `motion` in [CollabPositionIndicators](https://github.com/hashintel/hash/blob/main/packages/hash/frontend/src/blocks/page/CollabPositionIndicators.tsx) (it's an element that shows other users viewing a block) and that's because it was implemented before we switched to using MUI for our styling. 

We'd probably need to replace `motion.div` in CollabPositionIndicators with an MUI equivalent so we can remove framer from our dependencies. This will probably be done in a separate task

Reply:
Switched to `<Fade />` component in https://github.com/hashintel/hash/pull/914/commits/f8620af216a6e4b3ed382f0bb1c5fa4707d1b746